### PR TITLE
Metadata v14 pretty prints 

### LINKED ISSUE #179: Implement metadata v14 pretty prints
#178 introduces support for metadata v14. However, the new code doesn't feature nice prints for the metadata (yet).

Tasks:
- [x] implement sensible metadata prints
- [x] fix example_print_metadata
- [x] reactivate this example in GA.
- [ ] Consider #156

### PR DESCRIPTION
**Pull request by echevrier**
example-print-metadata is running as previously

* Closes #179

### CODE REVIEW

File: src/examples/example_print_metadata.rs [hunk 1]
```diff
@@ -38,10 +37,11 @@ fn main() {
     let meta = Metadata::try_from(api.get_metadata().unwrap()).unwrap();
 
     meta.print_overview();
-    meta.print_modules_with_calls();
-    meta.print_modules_with_events();
+    meta.print_pallets();
+    meta.print_pallets_with_calls();
+    meta.print_pallet_with_events();
     meta.print_modules_with_errors();
```
Comment (clangenb) [inline src/examples/example_print_metadata.rs:43]: Here you can also change modules to pallets. :)

**Review by clangenb — APPROVED** [review]
Looks good to me, only the minor typo :)

I agree not to tackle #156 yet.

File: src/examples/example_print_metadata.rs (hunk 1 above)
Comment (echevrier) [reply src/examples/example_print_metadata.rs:43]: Ouch ;)

**Comment by echevrier** [comment]
> Looks good to me, only the minor typo :)
> 
> I agree not to tackle #156 yet.

OK. Then I will close this task, after the merge.

**Pull request merged**